### ISSUE TITLE
MBP Late 2011 OS X 10.11.3

### ISSUE THREAD

**Issue by PerfectStory** [dokterdok/Continuity-Activation-Tool#269]
Doesn't appear to work on latest beta. I turned off the protection as instructed and when I restarted and opened the program again, it tells me to turn off the protection again which was already done in the first place. 

Ran the old Beta 3 Con tool, and had same results with the Beta 4 tool.

**Comment by heyitsruss** [comment]
I am also having this issue with a Early 2011 MBP running El Capitan version 10.11.2. I am also told to disable SIP even though the tool has verified that the SIP is already off

**Comment by razmoket13** [comment]
Same issue here : MBP early 2011, CAT was working perfectly on 10.11.1 (as for previous OS X versions)
I just updated to 10.11.2, and now CAT refuses to reactivate Continuity, it pretends SIP is on whereas I disabled it (recovery mode / terminal / csrutil disable).

**Comment by mobile4** [comment]
MBP 15' early 2011, Running 10.11.2 with the latest CAT. SIP is off. Everything works.

**Comment by jimicement** [comment]
MBP 15' late 2011 OSX 10.11.2 working with latest CAT except twice in the last two days have been unable to wake from sleep via bluetooth.  Opened lid and within 30 seconds bluetooth keyboard and touchpad connected.  SIP has been reenabled.

**Comment by mobile4** [comment]
jimicement. Do you mean wake up by external bluetooth keyboard?

**Comment by jimicement** [comment]
Correct.  I keep the lid closed since I have a thunderbolt display.  The internal keyboard is hardwired.

**Comment by mobile4** [comment]
I'm not sure but maybe you should use InsomniaX or NoSleep to disable sleep mode when the lid is closed. And setup bluetooth behavior in MBR's settings. I cannot test it myself cause I have no thunderbolt display. If you keep the lid opened the problem exists?

**Comment by mobile4** [comment]
Any successful CAT installation on final 10.11.3?

**Comment by jimicement** [comment]
MBP 15' late 2011 OSX 10.11.2 now 10.11.3

Updated to 10.11.3.
Disabled SIP.
Ran CAT.
Confirmed Bluetooth in System Report.
   Bluetooth Low Energy Supported: Yes
   Handoff Supported: Yes
   Instant Hot Spot Supported:	Yes
Confirmed Wi-Fi in System Report.
   Supported PHY Modes: 802.11 a/b/g/n
   Wake On Wireless: Supported
   AirDrop: Supported
Confirmed iPhone showing as 'Personal Hotspot' in Wi-Fi.
Enabled SIP.

**Comment by mobile4** [comment]
jimicement. Thank you for reply. Is it just confirmations from System Report? Or everything has been checked and it works?

**Comment by mobile4** [comment]
Macbook Pro, 15, early 2011, El Capitan 10.11.3. Airport card replaced. Everything works fine!

**Comment by sysfloat** [comment]
Just installed the final version of 10.11.3 and verything seems to work fine so far on my early 2011 MBP.
If it doesnt work for you, please post your CAT system diagnostics and the System Information.

**Comment by AbeEstrada** [comment]
Works on 10.11.3 from the App Store (not beta) on a MBP 15" 2010 with the BCM94331PCIEBT4CAX AirPort card

**Comment by mobile4** [comment]
Sometimes after wake up from sleep mode or rebooting bluetooth icon disappears from menu. If I turn it on in Settings it is Ok. But Airdrop doesn't work. To make it working I have to Pair Macbook and iPhone again in bluetooth settings. It looks like com.apple.Bluetooth.plist replaced or re-written on wake up or start up. How to solve it? Do you have com.apple.Bluetooth.plist.lockfile in Preferences folder of your system?

**Comment by mobile4** [comment]
Airdrop is very unstable. It works for a couple of days than suddenly stops. CAT diagnostics says everything fine. Mac system info also fine. Hand Off works. Only Airdrop feature which is most important for me fails. I tried to reinstall CAT several times. log of and log in to iCloud. It helps for 1-2 days. Any suggestion?

**Comment by jimicement** [comment]
No suggestions.  Airdrop, Handoff, and especially for me 'Personal Hotspot' fail from time to time.  Has that been the case since?

MBP 15' late 2011 OSX 10.11.3

In the last two weeks my BT/WiFi failed  3 times on reboot and I received the Continuity alert each time.  The first two times I disabled SIP and I was back.  Reenabling SIP soon would bring back the Continuity alert.  It's been a couple weeks now with no issue.  Before enabling SIP the last time I ran this from the CAT contitool.sh script:

sudo kextcache -system-caches

All good.  10.11.4 any day now...
